Block/Config.php

```diff
@@ -99,4 +99,30 @@ public function getLocale()
     {
         return $this->localeResolver->getLocale();
     }
+
+    /**
+     * @return string
+     */
+    public function getProductSku()
+    {
+        $product = $this->getProduct();
+        $children = $this->config->getUseChildSku() ? $product->getTypeInstance()->getUsedProducts($product) : [];
+
+        return count($children) > 0 ? array_values($children)[0]->getSku() : $product->getSku();
```

Comment:
If a configurable product does not have any children, I believe this code will cause an error as `array_values($children)[0]` will be null.